pkg/securitypolicy/securitypolicy.go

```diff
@@ -172,3 +181,14 @@ func (e EnvRules) MarshalJSON() ([]byte, error) {
 		Alias:  (*Alias)(&e),
 	})
 }
+
+func (em ExpectedMounts) MarshalJSON() ([]byte, error) {
+	type Alias ExpectedMounts
```

Comment:
just curious but why do we need to do this?

Reply:
In case anyone else has this same question: the answer is that this allows us to override the Length value when marshalling to the intended value instead of setting it by hand.